boards/arm/msp_exp432e401y_dk/support/MSP432E401Y.ccxml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What is this file for?

Reply:
The file is used by Code Composer Studio for XDS110 Debug Probe configuration (found on Ti SimpleLink Launchpads) .
Open OCD not using this file.